TypeDuck/OptionsView.swift

```diff
@@ -6,13 +6,11 @@ struct OptionsView: View {
         @EnvironmentObject private var context: AppContext
 
         private let options: [String] = [
-                "傳統漢字",
-                "傳統漢字・香港",
-                "傳統漢字・臺灣",
-                "简化字",
-                "半形數字",
-                "全形數字",
```

Comment:
This actually converts everything to fullwidth, not only numbers but also symbols and letters